core/lib/utils/benchmark.dart

```diff
@@ -0,0 +1,29 @@
+
+import 'package:core/utils/app_logger.dart';
+
+class _Benchmark {
```

Comment:
should run it only in debug or profile mode

Reply:
oke i will update it